vizro-core/docs/pages/user_guides/custom_charts.md

```diff
@@ -65,13 +69,18 @@ Building on the above, there are several routes one can take. The following exam
                 ),
             ],
             controls=[
-                vm.Filter(column="petal_width"),
+                vm.Parameter( # (1)!
+                    targets=["enhanced_scatter.hline"],
+                    selector=vm.Slider(min=2, max=5, step=1, value=3, title="Horizontal line"),
+                ),
             ],
         )
         dashboard = vm.Dashboard(pages=[page_0])
 
         Vizro().build(dashboard).run()
         ```
+
+        1.  Note how we are able to target *any custom argument* of our custom chart. Since there is complete flexibility in terms of what can be derived from such custom arguments this allows for a wide range of customization options by the dashboard user.
```

Comment:
Are there custom arguments? or are they just arguments?

```suggestion
        1.  Note how we can target *any argument* of our custom chart. Since there is complete flexibility regarding what can be derived from such arguments, the dashboard user has a wide range of customization options.
```